### DISCUSSION TITLE
How to get back to NnnExplorer after opening file

### DISCUSSION THREAD

**Discussion by Purple-Tentacle** [mcchrish/nnn.vim#165] [Q&A]
If open vim and then open nnn in Explorer mode (:NnnExplorer), I can chose a file, hit enter and it is opened in vim. Great.
Question: How to get back to nnn to open another file?

**Comment by rollf** [comment]
You can use the normal shortcuts to switch to the window again (say `<C-w>k` if the explorer window is above).
